KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/KeywordSearchIngestModule.java

```diff
@@ -632,10 +634,15 @@ private void indexFile(AbstractFile aFile, boolean indexContent) {
                 if (context.fileIngestIsCancelled()) {
                     return;
                 }
-                if (fileType.equals("application/octet-stream")) {
+                if (fileType.equals(MimeTypes.OCTET_STREAM)) {
                     extractStringsAndIndex(aFile);
                     return;
                 }
+                if (fileType.equals(MimeTypes.PLAIN_TEXT)) {
```

Comment:
I think we should now remove this because the same thing will now happen in extractTextAndIndex()  since TextExtractor is being returned from the Factory.  Keeping this out reduces a bit of complexity.

Reply:
Agreed, I went ahead and removed it.